test/exercises/Mandelbrot/MPlot.chpl

```diff
@@ -160,21 +161,24 @@ const header_size = 14;
   const offset_to_pixel_data = header_size + dib_header_size;
 
   // Write the BMP image header
-  outfile.writef("BM%<4u %<2u %<2u %<4u",
-                 size,
-                 0 /* reserved1 */,
-                 0 /* reserved2 */,
-                 offset_to_pixel_data);
-  
+  outfile.writef("BM");
+  outfile.writeBinary(size:uint(32), ioendian.little);
+  outfile.writeBinary(0:uint(16), ioendian.little); /* reserved1 */
```

Comment:
Does this lose the spaces between what you're writing?

Reply:
This is very interesting. I tried a sample program that wrote two binary integers using the format string `%|4i %|4i` and it didn't insert the space between them. I don't know if this is by design or a bug. The headers in MPlot.chpl don't have spaces so converting to `writeBinary` worked. But I need to investigate the space issue more.

Reply:
I think it was intentional but it is moot since we are deprecating it, right?